src/main/java/io/codis/jodis/RoundRobinJedisPool.java

```diff
@@ -114,6 +127,8 @@ public PooledObject(String addr, JedisPool pool) {
 
     private final String clientName;
 
+    private final Timer jedisPoolClosingTimer = new Timer();
```

Comment:
Usually it is better to use ScheduledExecutorService instead of Timer.

See this:
https://stackoverflow.com/questions/409932/java-timer-vs-executorservice

Reply:
Thanks for the link.